.github/workflows/build.yml

```
@@ -380,7 +400,7 @@ jobs:
           exit 1
       - name: Check Dependency Outcomes
         run: |
-          if [[ "${{ needs.validate.result }}" == "success" && "${{ needs.test.result }}" == "success" ]]; then
+          if [[ "${{ needs.validate.result }}" == "success" && ("${{ needs.configure.outputs.docChangeOnly }}" == "true" || "${{ needs.test.result }}" == "success") ]]; then
```

```suggestion
          if [[ "${{ needs.validate.result }}" == "success" && (${{ needs.configure.outputs.docChangeOnly }} || "${{ needs.test.result }}" == "success") ]]; then
```

is it possible to to like above, using system check for the value?